frontend/src/pages/admin/Template/index.vue

```diff
@@ -276,12 +290,18 @@ export default {
                     name: 'Admin Templates'
                 })
             } catch (err) {
+                if (err.response?.data) {
+                    alerts.emit(err.response.data.error, 'warning')
+                } else {
+                    alerts.emit('Unknown Error. Check logs.', 'warning')
```

Comment:
This is a potentially user-facing error, right? As far as I understand they wouldn't be able to see the logs. Is there other information we could surface to help them (and us) when they ultimately write into support if it keeps happening?

Reply:
Let's leave it as it is, other than adding some extra fluff words ("An unhandled error occurred, please try again or check the logs."), I can't think of a better suggestion right now!

Reply:
Another thing I forgot to mention... Once the validation is fixed on this form (see #1067) neither of the toast notifications will be seen unless there are exceptional circumstances.